packages/cardano-services/src/Asset/openApi.json

```diff
@@ -0,0 +1,102 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "UTxO",
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    },
+    "version": "1.0.0"
+  },
+  "paths": {
+    "/asset/get-token-metadata": {
+      "post": {
+        "summary": "Token Metadata",
+        "operationId": "getTokenMetadata",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/TokenMetadataRequest"
+              }
+            }
+          },
+          "required": true
+        },
+        "responses": {
+          "200": {
+            "description": "success operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/TokenMetadataResponse"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "invalid request",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object"
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "schemas": {
+      "TokenMetadata": {
+        "required": ["decimals", "desc", "icon", "name", "ticker", "url"],
```

Comment:
I'm not sure about this line.
Sorry @rhyslbw  and @mkazlauskas , could you please confirm which are the attributes we can trust will be for sure in the source token API? I know, this line validates the output of our http server, but the presence of these attributes strictly depends on the  presence in the token API.